lib/patientFinder/mongoid.yml

```diff
@@ -0,0 +1,34 @@
+development:
+  sessions:
+    default:
+      database: patid
```

Comment:
that's fine - even if we save the test records on the same collection (Cypress db records collection) it shouldn't be a a problem as the master records will not be affected in any way